### ISSUE TITLE
Getting error while importing a flow after exporting it

### ISSUE THREAD

**Issue by mohitsharma-22** [glific/glific#1785]
**Describe the bug**
Importing a flow throws the below error message.

**To Reproduce**
1. Go to FLows
2. Export the flow. save JSON file
3. change the name of the existing flow
4. Import the flow from the saved JSON file.

**Screenshots**
<img width="1016" alt="Screenshot 2021-09-23 at 10 17 02 AM" src="https://user-images.githubusercontent.com/16541548/134456402-3d75a9ad-065f-4297-92e1-2395bca61eef.png">

[TEST.json.zip](https://github.com/glific/glific/files/7215059/TEST.json.zip)

**Comment by pankaj-ag** [comment]
Great thank you. 

@mohitsharma-22 if you can share the use case of this? I mean when we need to import the flow and export it again on the same account that will help to fix the error.

**Comment by mohitsharma-22** [comment]
@pankaj-ag You are right there will not be any use case to export and import the flow in the same account. Expect if NGO has saved the flows as backup and needs for some case to import it again. Like activation of suspended accounts.
